scripts/linters/pre_commit_linter.py

```diff
@@ -81,7 +82,8 @@
 from . import other_files_linter  # isort:skip
 from . import python_linter  # isort:skip
 from .. import concurrent_task_utils  # isort:skip
-from .. import install_third_party_libs  # isort:skip
+if not feconf.OPPIA_IS_DOCKERIZED:
```

Comment:
Ideally, we are not planning to populate all the scripts with this flag, so as a tracking task remove this flag once running checks in a separate container.

Reply:
sure, on permenantly moving to Docker setup, the use of this check will be removed.